hadoop-ozone/cli-shell/src/main/java/org/apache/hadoop/ozone/shell/acl/GetAclHandler.java

```diff
@@ -19,19 +19,62 @@
 
 import java.io.IOException;
 import java.util.List;
+import java.util.stream.Collectors;
 import org.apache.hadoop.ozone.OzoneAcl;
 import org.apache.hadoop.ozone.client.OzoneClient;
 import org.apache.hadoop.ozone.security.acl.OzoneObj;
+import picocli.CommandLine.Option;
 
 /**
  * Get ACLs.
  */
 public abstract class GetAclHandler extends AclHandler {
 
+  @Option(names = {"--json"},
+      description = "Format output as JSON",
```

Comment:
Let's add more explanation in description. 

Format output as JSON. Default is true. Use --json=false to turn off output JSON format.

Reply:
Thanks @ChenSammi for pointing this out.
I will add more explanation in the next commit.